testgrid/config/config.yaml

```diff
@@ -1601,6 +1601,71 @@ dashboards:
     base_options: 'include-filter-by-regex=pkg/kubectl%7COverall'
     description: 'kubectl e2e tests in gke-master-1.6-gci-kubectl-skew-serial'
 
+- name: sig-storage
+  dashboard_tab:
+  - name: gce
+    test_group_name: ci-kubernetes-e2e-gce
+    base_options: 'include-filter-by-regex=Volume'
+    description: 'storage gce e2e tests for master branch'
+  - name: gke
+    test_group_name: ci-kubernetes-e2e-gke
+    base_options: 'include-filter-by-regex=Volume'
+    description: 'storage gke e2e tests for master branch'
+  - name: gce-1.6
+    test_group_name: ci-kubernetes-e2e-gce-release-1.6
+    base_options: 'include-filter-by-regex=Volume'
+    description: 'storage gce e2e tests for 1.6 branch'
+  - name: gke-1.6
+    test_group_name: ci-kubernetes-e2e-gke-release-1.6
+    base_options: 'include-filter-by-regex=Volume'
+    description: 'storage gke e2e tests for 1.6 branch'
+  - name: gce-1.5
+    test_group_name: ci-kubernetes-e2e-gce-release-1.5
+    base_options: 'include-filter-by-regex=Volume'
+    description: 'storage gce e2e tests for 1.5 branch'
+  - name: gke-1.5
+    test_group_name: ci-kubernetes-e2e-gke-release-1.5
+    base_options: 'include-filter-by-regex=Volume'
+    description: 'storage gke e2e tests for 1.5 branch'
+  - name: gce-flaky
+    test_group_name: ci-kubernetes-e2e-gce-flaky
+    base_options: 'include-filter-by-regex=Volume'
+    description: 'storage gce flaky e2e tests for master branch'
+  - name: gke-flaky
+    test_group_name: ci-kubernetes-e2e-gke-flaky
+    base_options: 'include-filter-by-regex=Volume'
+    description: 'storage gke flaky e2e tests for master branch'
+  - name: gce-slow
+    test_group_name: ci-kubernetes-e2e-gce-slow
+    base_options: 'include-filter-by-regex=Volume'
+    description: 'storage gce slow e2e tests for master branch'
+  - name: gke-slow
+    test_group_name: ci-kubernetes-e2e-gke-slow
+    base_options: 'include-filter-by-regex=Volume'
+    description: 'storage gke slow e2e tests for master branch'
+  - name: gce-serial
+    test_group_name: ci-kubernetes-e2e-gce-serial
+    base_options: 'include-filter-by-regex=Volume'
+    description: 'storage gce serial e2e tests for master branch'
+  - name: gke-serial
+    test_group_name: ci-kubernetes-e2e-gke-serial
+    base_options: 'include-filter-by-regex=Volume'
+    description: 'storage gke serial e2e tests for master branch'
+  - name: unit-integration-cmd
+    test_group_name: ci-kubernetes-test-go
+    # test-cmd, test-integration do not produce xml data right now
```

Comment:
Please don't just blindly copy stuff from sig-cli. Nothing about test-cmd has any relevance to storage.

I am not sure whether test-integration has any relevance